packages/playground/src/dashboard/components/create_farm.vue

```diff
@@ -45,7 +45,14 @@
             </form-validator>
           </v-card-text>
           <v-card-actions class="justify-end my-1 mr-2">
-            <v-btn color="anchor" @click="showDialogue = false">Close</v-btn>
+            <v-btn
+              color="anchor"
+              @click="
+                showDialogue = false;
+                $emit('update:name', '');
```

Comment:
why do we pass the name to the parent it is not used as i see, i think this behavior need some refactor, no need to bind the farm name here

Reply:
it's just a reset for farm name on close

Reply:
we can just remove the name prop, it will reset the farm name.
we are not using the name on the parent component

Reply:
When I did it disappeared on blur before submit / close.

Reply:
this is not related to props, this related to the dialog component, u can try to add [persistent](https://vuetifyjs.com/en/api/v-dialog/#props-persistent) to the dialog component

Reply:
done